sputnikdao2/src/types.rs

```diff
@@ -64,3 +67,10 @@ impl Action {
         format!("{:?}", self)
     }
 }
+
+pub fn convert_old_to_new_token(old_account_id: &OldAccountId) -> Option<AccountId> {
```

Comment:
@ctindogaru thoughts about commenting a flag about this?
Would like to see more comments about why this was added, and details of when it can be removed.

Reply:
I think that's a very good suggestion. Will write a comment about it.